### PR TITLE
OpenMW-CS: Fix minimap corruption

### PR DESCRIPTION
**Pull request by unelsson**
fix bug [5177](https://gitlab.com/OpenMW/openmw/issues/5177)

mWnam wasn't copied in custom copy constructor

### CODE REVIEW

File: components/esm/loadland.cpp [hunk 1]
```diff
@@ -340,6 +342,10 @@ namespace ESM
         std::swap (mContext, land.mContext);
         std::swap (mDataTypes, land.mDataTypes);
         std::swap (mLandData, land.mLandData);
+        for (int i = 0; i < LAND_GLOBAL_MAP_LOD_SIZE; ++i)
+        {
+            std::swap (mWnam[i], land.mWnam[i]);
```
Comment (Capostrophic) [inline components/esm/loadland.cpp]: Equally sized arrays can be swapped without a loop going through all of their elements.

Comment (unelsson) [reply components/esm/loadland.cpp]: I thought it was only std::arrays, so I used this method during bug-hunt to keep track of values. Anyway, fixed in 2fdaacf . Also tested that it really does work.

**Comment by Capostrophic** [comment]
Update the changelog pls.

**Comment by unelsson** [comment]
> Update the changelog pls.

Yeaah.. Done. And conflict.

**Pull request merged**